### PR TITLE
[doc] Improve documentation for run search

### PR DESCRIPTION
**Pull request by mihran113**
- add usage of `created_at`, `finalized_at`, `duration`, `metrics` properties

### CODE REVIEW

**Review by alberttorosyan — APPROVED** [review]
lgtm

File: docs/source/using/search.md [hunk 1]
```diff
@@ -109,6 +114,59 @@ run.learning_rate in [0.0001, 0.005]
...
+|-----|------------|---------|
+| `run_1 <hash=a32c910>` | <table><thead> <tr> <th>learning_rate</th><th>batch_size</th> </tr> </thead>  <tbody> <tr> <th>0.001</th><th>32</th> </tr> </tbody></table> | <table><thead> <tr><th>loss { "subset":"train" } </th><th> loss { "subset":"test" } </th></tr></thead> <tbody><tr><th> 10 </th><th> 10 </th><th></tbody> </table>|
+| `run_2 <hash=a32c911>` | <table><thead> <tr> <th>learning_rate</th><th>batch_size</th> </tr> </thead>  <tbody> <tr> <th>0.0007</th><th>64</th> </tr> </tbody></table> | <table><thead> <tr><th>loss { "subset":"train" } </th><th> loss { "subset":"test" } </th><th> accuracy {} </th></tr></thead> <tbody><tr><th> 20 </th><th> 20 </th><th> 0.2 </th></tr></tbody> </table>|
+
+4. Search runs where `duration` is smaller than 10s (to filter out failed runs for example)
+```python
+run.duration < 10
+```
+
+This will yield an empty result on our test runs, but will have a use case on real life experiments to filter the runs that finished under 10 seconds and then archive them as they've probably crashed or were stopped.
```
Comment (gorarakelyan) [inline docs/source/using/search.md:150]: will it yield an empty result or will all the runs be returned?

Comment (mihran113) [reply docs/source/using/search.md:150]: Yup, well noticed, I initially wrote the text for `run.duration > 10` query, forgot to change it 😀

**Review by gorarakelyan — APPROVED** [review]
lgtm

**Pull request merged**